src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/UserService.kt

```diff
@@ -72,6 +72,8 @@ class UserService(
     return userRepository.findByNameContainingIgnoreCase(name)
   }
 
+  fun findById(id: UUID) = userRepository.findById(id)
```

Comment:
I think Kotlin neatly removes the need for Optionals given how well it deals with nulls, so maybe here we could use findByIdOrNull instead?